NDTensors/test/ITensors/TestITensorDMRG/dmrg.jl

```diff
@@ -0,0 +1,37 @@
+using Test
+using ITensors
+using NDTensors
+
+function test_dmrg(elt, N::Integer, dev::Function, cut::Float64, no::Float64)
+  # Create N spin-one degrees of freedom
+  sites = siteinds("S=1", N)
+
+  # Input operator terms which define a Hamiltonian
+  os = OpSum()
+  for j in 1:(N - 1)
+    os += "Sz", j, "Sz", j + 1
+    os += 0.5, "S+", j, "S-", j + 1
+    os += 0.5, "S-", j, "S+", j + 1
+  end
+
+  # Convert these terms to an MPO tensor network
+  H = dev(MPO(elt, os, sites))
+
+  # Create an initial random matrix product state
+  psi0 = dev(randomMPS(elt, sites;))
```

Comment:
```suggestion
  psi0 = dev(randomMPS(elt, sites; linkdims=4))
```
Good to test constructing an MPS with a nontrivial bond dimension.